docs/FAQ.md

```diff
@@ -48,10 +48,9 @@ source candidates. (And we'll probably end up writing some brand new ones, too.)
 ### What's the roadmap for Backstage?
 
 We envision three phases, which you can learn about in
-[our project roadmap](https://github.com/backstage/backstage#project-roadmap).
-Even though the open source version of Backstage is relatively new compared to
-our internal version, we have already begun work on various aspects of all three
-phases. Looking at the
+[our project roadmap](overview/roadmap). Even though the open source version of
```

Comment:
One item I forgot, is if these should be defined to work in raw code and then let Mkdocs rewrite it?  e.g., should this instead be `overview/roadmap.md` and would mkdocs realign it properly?